content/developer/addons/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Addons
+tags:
+- Developers
+- Addons
+- Plugins
+- Addons
+- Themes
+category: developer
+menu:
+  developer:
+    weight: 11
+    identifier: addons
+aliases:
+- /developers/plugins
+- /developer/plugins
+---
+
+Vanilla is a very customizable, flexible platform because there are many ways to hook into it without ever modifying its code. There are normal vanilla [addons](/developer/addons/addon-quickstart) and [themes](/developer/addons/theme-quickstart). Themes and addons are very similar, but serve different purposes.
+
+## Addons and Plugins and Themes - Oh My!
```

Comment:
Since we're deleting the old plugins pages, it might be good to have a little blurb on pages being redirected about the change...